### PR TITLE
Expose tissue_name through cerberus_fp and pinery_ius

### PR DESCRIPTION
**Pull request by apmasell**
File provenance returns a `Collection` for the parents. It should be a collection of one, as is the custom of having things turn to set in file provenance.

### CODE REVIEW

File: plugin-niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/cerberus/IUSUtils.java [hunk 1]
```diff
@@ -68,8 +68,22 @@ public static long parseLong(String input) {
...
+        .findFirst()
+        .orElse("");
+  }
+
   private static final Imyhat ATTR_TYPE = Imyhat.tuple(Imyhat.STRING, Imyhat.STRING.asList());
+  private static final Pattern COLON = Pattern.compile(":");
   private static final Pattern LANE_NUMBER = Pattern.compile("^.*_(\\d+)$");
+  private static final Pattern TISSUE_REGEX =
+      Pattern.compile(
+          "^([A-Z0-9]{3,10}_\\d{4,}_\\d{2,}|[A-Z0-9]{3,10}_\\d{4,}_[A-Zn][a-z]_[A-Zn]_(nn|\\d{2})_\\d+_\\d+)$");
```
Comment (djcooke) [inline plugin-niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/cerberus/IUSUtils.java]: Sorry, looks like I made a mistake in the regex I sent you. The last underscore should actually be a dash instead

Comment (apmasell) [reply plugin-niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/cerberus/IUSUtils.java]: Fixed.

File: plugin-niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/cerberus/IUSUtils.java [hunk 2]
```diff
@@ -68,8 +68,22 @@ public static long parseLong(String input) {
...
+        .findFirst()
+        .orElse("");
+  }
+
   private static final Imyhat ATTR_TYPE = Imyhat.tuple(Imyhat.STRING, Imyhat.STRING.asList());
+  private static final Pattern COLON = Pattern.compile(":");
   private static final Pattern LANE_NUMBER = Pattern.compile("^.*_(\\d+)$");
+  private static final Pattern TISSUE_REGEX =
+      Pattern.compile(
+          "^([A-Z0-9]{3,10}_\\d{4,}-\\d{2,}|[A-Z0-9]{3,10}_\\d{4,}_[A-Zn][a-z]_[A-Zn]_(nn|\\d{2})_\\d+_\\d+)$");
```
Comment (djcooke) [inline plugin-niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/cerberus/IUSUtils.java]: I meant the *last* last underscore (in old scheme between times received and tube number):

```
^([A-Z0-9]{3,10}_\\d{4,}_\\d{2,}|[A-Z0-9]{3,10}_\\d{4,}_[A-Zn][a-z]_[A-Zn]_(nn|\\d{2})_\\d+-\\d+)$
```

Comment (apmasell) [reply plugin-niassa+pinery/src/main/java/ca/on/oicr/gsi/shesmu/cerberus/IUSUtils.java]: Yikes. Fixed. :dizzy:

**Pull request merged**